src/providers/fault-count/cat-a-mod2/__tests__/fault-count.cat-a-mod2.spec.ts

```diff
@@ -40,4 +41,22 @@ describe('FaultCountAM2Helper', () => {
     const temp = FaultCountAM2Helper.getSafetyAndBalanceFaultCountCatAM2(safetyAndBalanceMock3FaultsSafetyAndBalance);
     expect(temp).toEqual(output);
   });
+
+  describe('getDangerousFaultSumCountCatAM2', () => {
+    it('should return the dangerous faults count for cat AM2', () => {
+      expect((FaultCountAM2Helper as any).getDangerousFaultSumCountCatAM2(catAM2TestDataStateObject)).toBe(1);
```

Comment:
why does this need to be cast?

Reply:
I used other tests as a base for this. If they don't need casting I will remove it.

Reply:
Removed casting